app/controllers/api/teams.php

```diff
@@ -208,7 +238,7 @@
     ->label('sdk.auth', [APP_AUTH_TYPE_SESSION, APP_AUTH_TYPE_KEY, APP_AUTH_TYPE_JWT])
     ->label('sdk.namespace', 'teams')
     ->label('sdk.method', 'update')
```

Comment:
Should we change the SDK method too?

Reply:
good question, that would end up in a breaking change 🤔

Reply:
What do you think @eldadfux?

Reply:
actually not a breaking change, since the url doesnt change. so will work with old sdks